bundles/io/org.eclipse.smarthome.io.rest/src/main/java/org/eclipse/smarthome/io/rest/internal/filter/CorsFilter.java

```diff
@@ -45,10 +45,13 @@
     private static final String HTTP_POST_METHOD = "POST";
     private static final String HTTP_GET_METHOD = "GET";
     private static final String HTTP_OPTIONS_METHOD = "OPTIONS";
+    
+    private static final String CONTENT_TYPE_HEADER = "content-type";
```

Comment:
You're right. Whenever I do XHR request, I write like this `Content-Type`. Although, browser and server does not care about casing and I got that idea from the log. 
![cors-preflight](https://cloud.githubusercontent.com/assets/3100980/12608762/fcfc4cb4-c503-11e5-9b0f-be146621116f.png)

So I used the non-cased version of the string. Both are right but I felt safer with this.